src/main/java/org/opensearch/knn/quantization/models/quantizationOutput/BinaryQuantizationOutput.java

```diff
@@ -63,4 +63,16 @@ public boolean isPrepared(int vectorLength) {
     public byte[] getQuantizedVector() {
         return quantizedVector;
     }
+
+    /**
+     * Returns a copy of the quantized vector.
+     *
+     * @return a copy of the quantized vector byte array.
+     */
+    @Override
+    public byte[] getQuantizedVectorCopy() {
```

Comment:
[Not a blocker]
Also lets have some unit test for `getQuantizedVector` and `getQuantizedVectorCopy` functions to ensure that same reference is returned when `getQuantizedVector` is called and different is returned when `getQuantizedVectorCopy`.